lib/stores/GCSDataStore.js

```diff
@@ -2,7 +2,7 @@
 
 const DataStore = require('./DataStore');
 const File = require('../models/File');
-const gcloud = require('google-cloud');
+const storage = require('@google-cloud/storage');
```

Comment:
Do you mind rebasing from master in order to remove these changes from #90 to isolate S3?

Reply:
not at all, will do it asap.

Reply:
done!